web_menu_collapsible/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Collapsible menu
+================
+
+This module makes all menus collapsible for all users.
```

Comment:
No, I mean before clicking in them :wink: 

Do you have a better sentence for it?